components/sup/doc/http_gateway_census_schema.json

```diff
@@ -0,0 +1,316 @@
+          "update_leader_id"
+        ],
+        "properties": {
+          "changed_service_files": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "description": "The names of the service files that have changed"
+          },
+          "election_status": {
+            "type": "string",
+            "description": "The status of the current election, if any"
+          },
+          "leader_id": {
+            "type": ["string", "null"],
+            "description": "The member ID of the election leader, if there is one"
+          },
+          "local_member_id": {
+            "type": "string",
+            "description": "The local member ID"
+          },
+          "population": {
+            "type": "object",
+            "description": "The census population, keyed by member ID",
+            "additionalProperties": {
+              "type": "object",
+              "description": "The census member",
+              "properties": {
+                "alive": {
+                  "type": "boolean",
+                  "description": "Whether the member is alive"
+                },
+                "application": {
+                  "type": "null",
+                  "description": "The application portion of a service group specification"
+                },
+                "cfg": {
+                  "type": "object",
+                  "description": "The configuration this member is currently exporting"
+                  "additionalProperties": {
+                    "type": ["boolean", "string", "integer", "array", "object"]
+                  }
+                },
+                "confirmed": {
+                  "type": "boolean",
+                  "description": "Whether this member is confirmed dead/unreachable, from a network perspective"
+                },
+                "departed": {
+                  "type": "boolean",
+                  "description": "Whether this member has been departed from the ring"
+                },
+                "election_is_finished": {
+                  "type": "boolean",
+                  "description": "Whether a leader election for this service has finished"
+                },
+                "election_is_no_quorum": {
+                  "type": "boolean",
+                  "description": "Whether there is quorum for a leader electio for this service"
+                },
+                "election_is_running": {
+                  "type": "boolean",
+                  "description": "Whether there is a leader election running for this service"
+                },
+                "environment": {
+                  "type": "null",
+                  "description": "The environment portion of a service group specification",
+                  "additionalProperties": {
+                    "type": ["null", "boolean", "string", "integer", "array", "object"]
+                  }
```

Comment:
Isn't `environment` just a String?

Reply:
Yeah, you're right.  I don't know how I missed that, thanks.